snuba/cli/replacer.py

```diff
@@ -55,7 +55,7 @@ def replacer(replacements_topic, consumer_group, bootstrap_server, clickhouse_ho
 
     metrics = util.create_metrics(
```

Comment:
This function is not covered. I don't think this is high risk, considering that the `ReplacerWorker` is heavily covered. The main concern here would be if `create_metrics` didn't actually return a `Metrics` instance, or if the constant tags were applied incorrectly. The constant tags should be able to be easily checked with unit tests.